script.js

```diff
@@ -324,8 +407,8 @@ require(['jquery','core/str'], function ($, Str)
             }
 
             function open($dir, visible) {
-                var pix = icon[visible ? 'dir-open' : 'dir-closed'].pix;
-                $dir.find('> div img').first().attr('src', M.util.image_url(pix));
+                var iconElement = icon[visible ? 'dir-open' : 'dir-closed'].iconClass;
```

Comment:
It is bad practice to do logic inside an array - either do:

var iconElement = visible ? icon['dir-open'] : icon['dir-closed']

or:

var iconIndex = visible ? 'dir-open' : 'dir-closed';
var iconElement = icon[iconIndex]; // or just use it as icon[iconIndex] later on.